src/components/MovieCatalog/Filter/index.jsx

```diff
@@ -1,66 +1,65 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
 import PropTypes from 'prop-types';
 
 import './style.scss';
 import Select from '../../Select';
 
-class Filter extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      filter: 'all',
-      sort: 'release date'
-    };
-  }
+const Filter = ({ genres, options }) => {
+  const [filter, setFilter] = useState('all');
+  const [sort, setSort] = useState('release date');
 
-  handleChange = (evt) => {
-    this.setState({
-      sort: evt.target.value,
-    });
-    console.log(evt.target.value);
-  }
+  const handleChange = useCallback(
+    (evt) => {
+      setSort(evt.target.value);
+      console.log(evt.target.value);
+    },
+    [sort],
```

Comment:
This dependency is unnecessary 

Consider installing [eslint-plugin-react-hooks](https://www.npmjs.com/package/eslint-plugin-react-hooks)

Same thing for filter